### PR TITLE
Radical

### PR DESCRIPTION
**Pull request by argvsc47**
new theme

### CODE REVIEW

File: src/theme.ts [hunk 1]
```diff
@@ -311,6 +311,30 @@ export const COLORS: {[name: string]: Theme} = {
...
+    S_RANK_BASE: "#ffce32",
+    S_RANK_SHADOW: "#ffce32",
+    S_RANK_TEXT: "#f8d847",
+    A_RANK_BASE: "#06b86c",
+    A_RANK_SHADOW: "#06b86c",
+    A_RANK_TEXT: "#f8d847",
+    B_RANK_BASE: "#00006c",
+    B_RANK_SHADOW: "#00006c",
+    B_RANK_TEXT: "#f8d847",
+    DEFAULT_BASE: "#8be9fd",
```
Comment (ryo-ma) [inline src/theme.ts:334]: You made a mistake about the field name.
DEFAULT_RANK_XXX is required. Not DEFAULT_XXX.

**Comment by ryo-ma** [comment]
@argvsc47 Please fix it.

File: src/theme.ts (hunk 1 above)
Comment (argvsc47) [reply src/theme.ts:334]: ah you're right

Comment (argvsc47) [reply src/theme.ts]: i've fixed it

**Comment by ryo-ma** [comment]
スクリーンショット 2021-04-26 20 34 59

I can not see the rank bar. 
And the B rank trophy is too dark. The rank text is too light.
Please adjust colors or change colors.

File: src/theme.ts (hunk 1 above)
Comment (argvsc47) [reply src/theme.ts]: how about now

**Comment by argvsc47** [comment]
how about now ?

**Comment by ryo-ma** [comment]
@argvsc47 
Is there a color palette for the radical theme?
I want to little bit adjust the colors.

**Comment by argvsc47** [comment]
i mean there is no palette but feel free as long you don't change it too much :D

**Comment by ryo-ma** [comment]
@argvsc47 I have adjusted the colors. I referenced a radical-vscode color theme(https://github.com/DHedgecock/radical-vscode)
How about what do you think?

スクリーンショット 2021-04-29 19 51 54

**Comment by argvsc47** [comment]
Cool, this is nice a think we can close this PR now :D

**Comment by RayhanADev** [comment]
Kind of reminds me of darkhub.

**Comment by ryo-ma** [comment]
@RayhanADev It's probably right. If I fix the theme, It might similar color. So difficult 😅

**Comment by RayhanADev** [comment]
@ryo-ma yeah I figure xD

**Pull request merged**